build.gradle

```diff
@@ -38,28 +38,13 @@ allprojects {
         mavenLocal()
         jcenter() {
             content {
-
                 // Remove when https://github.com/google/ExoPlayer/issues/5246 is closed
                 includeGroup("com.google.android.exoplayer")
 
-                // Remove when https://github.com/facebook/flipper/issues/1775 is closed
-                includeGroup("com.facebook.flipper")
-
                 includeModule("me.relex", "photodraweeview")
```

Comment:
This library has an open issue now too https://github.com/ongakuer/PhotoDraweeView/issues/81